crates/accelerate/src/error_map.rs

```diff
@@ -41,6 +40,7 @@ pub struct ErrorMap {
 #[pymethods]
 impl ErrorMap {
     #[new]
+    #[pyo3(text_signature = "(num_qubits, num_edges, /")]
     fn new(size: Option<usize>) -> Self {
```

Comment:
This one doesn't match its signature.

Reply:
Lol, I just moved the line but yeah it's not correct. I'll fix it now